Python/fhirreference.py

```diff
@@ -4,13 +4,18 @@
 #  Subclassing FHIR's reference to add resolving capabilities
 
 import logging
+import re
+from urllib.parse import urlparse
```

Comment:
First commit in this PR states "[Python] Restore compatibility with Python 2" and here you probably broke it again.